apps/internal/exported/exported.go

```diff
@@ -12,3 +12,23 @@ type AssertionRequestOptions struct {
 	// TokenEndpoint is the intended token endpoint. Used as the assertion's "aud" claim.
 	TokenEndpoint string
 }
+
+// TokenProviderParameters is the authentication parameters passed to token providers
+type TokenProviderParameters struct {
```

Comment:
Consider naming this `AppTokenProvider...` since it only works for client_credential flow and not for user flows (auth_code, obo etc.)